test/letters/actions/letters.unit.spec.js

```diff
@@ -359,49 +370,61 @@ describe('getBenefitSummaryOptions', () => {
   });
 });
 
-describe.skip('getLetterPdf', () => {
+describe('getLetterPdf', () => {
   beforeEach(setup);
   afterEach(teardown);
 
-  // const civilSLetter = {
-  //   letterName: 'Civil Service Preference Letter',
-  //   letterType: LETTER_TYPES.civilService,
-  //   letterOptions: {
-  //     // Opts only relevant for BSL but ATM required in every download link
-  //     militaryService: true,
-  //     monthlyAward: true,
-  //     serviceConnectedEvaluation: true,
-  //     chapter35Eligibility: true,
-  //     serviceConnectedDisabilities: true
-  //   }
-  // };
-
-  // const benefitSLetter = {
-  //   letterName: 'Benefit Summary Letter',
-  //   letterType: LETTER_TYPES.benefitSummary,
-  //   letterOptions: {
-  //     // Opts only relevant for BSL but ATM required in every download link
-  //     militaryService: true,
-  //     monthlyAward: true,
-  //     serviceConnectedEvaluation: true,
-  //     chapter35Eligibility: true,
-  //     serviceConnectedDisabilities: true
-  //   }
-  // };
-
-  it('sets up fetch for benefit summary letter', () => {});
-
-  it('sets up fetch for non-benefit-summary letters', () => {});
-
-  it('dispatches PENDING action when download initiated', () => {});
-
-  it('handles ie10 stuff', () => {});
-
-  it('downloads stuff conditionally', () => {});
```

Comment:
Could you clarify why you didn't implement some of these tests? I know some of them are pretty hard to write tests for, but was that the only reason?

Reply:
Eh, I thought that most of this stuff ended up being duplicative. Looking at it again today, I think it may still be worth testing IE10 as well as DOWNLOAD_SUCCESS for non-BSL types.

Reply:
Added those two tests if you want to give it another look. Otherwise I'll merge this afternoon.